torch/ao/quantization/pt2e/qat_utils.py

```diff
@@ -694,9 +710,29 @@ def _fold_conv_bn_qat_helper(m: GraphModule, is_cuda: bool) -> GraphModule:
     m.recompile()
     _remove_extra_dequantize(m)
 
-    # Step (2): Fold BN weights into conv
+    # Step (2): Copy over metadata from original subgraph
+    for r in replacements:
+        (
+            replacement_conv_node,
+            replacement_bn_node,
+            replacement_getitem_node,
+            replacement_relu_node,
+        ) = _get_conv_bn_getitem_nodes(r.replacements)
+
+        for original_node in _filter_nodes_map(r.nodes_map).values():
```

Comment:
Andrew, why do you need to copy the metadata?

Reply:
This is for lowering models trained by QAT. We found that the `source_fn` was not preserved and all convs in the model ended up being in the same partition `conv2d`, but they should be in different partitions